src/transformers/utils/dummy_flax_objects.py

```diff
@@ -222,6 +222,74 @@ def from_pretrained(self, *args, **kwargs):
         requires_backends(self, ["flax"])
 
 
+class FlaxBigBirdForMaskedLM:
+    def __init__(self, *args, **kwargs):
+        requires_backends(self, ["flax"])
+
+    @classmethod
+    def from_pretrained(self, *args, **kwargs):
```

Comment:
Side note, we fixed those to use `cls` instead of self, so this PR will need a rebase and a new `make fix-copies` once #12097 is merged.

Reply:
done.